src/lualib/Error.ts

```diff
@@ -22,6 +24,10 @@ function getErrorStack(constructor: () => any): string | undefined {
 
     if (_VERSION.includes("Lua 5.0")) {
         return debug.traceback(`[Level ${level}]`);
+    } else if (_VERSION === "Lua 5.1") {
```

Comment:
Guess we should use `.includes` here too, to trick whatever type nonsense is going on with the version-specific types

Reply:
I was going by the declaration which only seems to have 5.1, I guess includes is used to catch all of the 5.0 subversions. I think I just need to include another `@ts-ignore` here. This one is my bad since I changed this last minute without running another build.